scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -16,47 +16,47 @@ object ScaladocParser {
 
   private val numberOfSupportedHeadingLevels = 6
 
-  private def hspace[_: P] = CharIn("\t\r ")
-  private def hspacesMin[_: P](min: Int) = P(CharsWhileIn("\t\r ", min))
-  private def hspaces0[_: P] = hspacesMin(0)
-  private def hspaces1[_: P] = hspacesMin(1)
-  private def hspacesMinWithLen[_: P](min: Int): P[Int] =
+  private def hspace[$: P] = CharIn("\t\r ")
```

Comment:
Looks like the previous breaks in Scala 3 and it's also changed in the examples of fastparse https://com-lihaoyi.github.io/fastparse/